src/realm-settings/DefaultGroupsTab.tsx

```diff
@@ -0,0 +1,228 @@
+import { KeycloakSpinner } from "../components/keycloak-spinner/KeycloakSpinner";
+import { useHelp } from "../components/help-enabler/HelpHeader";
+
+export const DefaultsGroupsTab = () => {
+  const { t } = useTranslation("realm-settings");
+
+  const [isKebabOpen, toggleKebab] = useToggle();
+  const [isGroupPickerOpen, toggleGroupPicker] = useToggle();
+  const [defaultGroups, setDefaultGroups] = useState<GroupRepresentation[]>();
+  const [selectedRows, setSelectedRows] = useState<GroupRepresentation[]>([]);
+
+  const [key, setKey] = useState(0);
+  const [load, setLoad] = useState(0);
+  const reload = () => setLoad(load + 1);
+
+  const adminClient = useAdminClient();
+  const { realm } = useRealm();
+  const { addAlert, addError } = useAlerts();
+  const { enabled } = useHelp();
+
+  useFetch(
+    () => adminClient.realms.getDefaultGroups({ realm }),
+    (groups) => {
+      setDefaultGroups(groups);
+      setKey(key + 1);
+    },
+    [load]
+  );
+
+  const loader = () => Promise.resolve(defaultGroups!);
+
+  const removeGroup = async () => {
+    try {
+      for (const group of selectedRows) {
+        await adminClient.realms.removeDefaultGroup({
+          realm,
+          id: group.id!,
+        });
+      }
+      addAlert(
+        t("groupRemove", { count: selectedRows.length }),
+        AlertVariant.success
+      );
+      setSelectedRows([]);
+    } catch (error) {
+      addError("realm-settings:groupRemoveError", error);
+    }
+    reload();
+  };
+
+  const addGroups = async (groups: GroupRepresentation[]) => {
+    try {
+      for (const group of groups) {
+        await adminClient.realms.addDefaultGroup({
+          realm,
+          id: group.id!,
+        });
+      }
+      addAlert(
+        t("defaultGroupAdded", { count: groups.length }),
+        AlertVariant.success
+      );
+    } catch (error) {
+      addError("realm-settings:defaultGroupAddedError", error);
+    }
+    reload();
+  };
+
+  const [toggleRemoveDialog, RemoveDialog] = useConfirmDialog({
+    titleKey: t("removeConfirmTitle", { count: selectedRows.length }),
+    messageKey: t("removeConfirm", { count: selectedRows.length }),
+    continueButtonLabel: "common:delete",
+    continueButtonVariant: ButtonVariant.danger,
+    onConfirm: removeGroup,
+  });
+
+  if (!defaultGroups) {
+    return <KeycloakSpinner />;
+  }
+
+  return (
+    <>
+      <RemoveDialog />
+      {isGroupPickerOpen && (
+        <GroupPickerDialog
+          type="selectMany"
+          text={{
+            title: "realm-settings:addDefaultGroups",
+            ok: "common:add",
+          }}
+          onConfirm={(groups) => {
+            addGroups(groups);
+            toggleGroupPicker();
+          }}
+          onClose={toggleGroupPicker}
+        />
+      )}
+      {enabled && (
+        <Popover
+          bodyContent={
+            <Trans i18nKey="realm-settings-help:defaultGroups">
+              Default groups allow you...
```

Comment:
Instead of adding text here you can also add an empty placeholder.
```suggestion
              {" "}
```